src/community/json-ld/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.geoserver</groupId>
+        <artifactId>community</artifactId>
+        <version>2.17-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.geoserver.community</groupId>
+    <artifactId>gs-json-ld</artifactId>
+    <version>2.17-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>json-ld Module</name>
```

Comment:
Additional details expected (description,...) and a developers section listing yourself as module maintainer

Reply:
I think we still need to list you as module maintainer?

Reply:
fixed